keps/sig-node/compute-device-assignment.md

```diff
@@ -13,10 +13,10 @@ approvers:
   - "@sig-node-leads"
 editor: "@dashpole"
 creation-date: "2018-07-19"
-last-updated: "2019-04-30"
+last-updated: "2020-07-06"
 status: implementable
 ---
-# Kubelet endpoint for device assignment observation details 
+# Kubelet endpoint for device assignment observation details
```

Comment:
In this KEP we are enabling support for obtaining information about kubelet's assignment of cpus to containers in addition to devices. I think we should update the title of this KEP to reflect that too. From the current title it seems to be only catering to devices.

Reply:
I think, yes ) I'll update.